### ISSUE TITLE
Deprecate swagger-codegen in favor openapi generator?

### ISSUE THREAD

**Issue by chenrui333** [meetup/rules_openapi#27]
> In May 2018, myself and other top contributors (40+) have decided to fork Swagger Codegen to maintain a community-driven version called "OpenAPI Generator" (https://openapi-generator.tech), which supports both OpenAPI/Swagger spec v2 and v3.

> For the reasons behind the fork, please refer to the Q&A: https://github.com/OpenAPITools/openapi-generator/blob/master/docs/qna.md. For migration, please refer to https://github.com/OpenAPITools/openapi-generator/blob/master/docs/migration-from-swagger-codegen.md

**Comment by chenrui333** [comment]
Looks like there is already one on this, https://github.com/OpenAPITools/openapi-generator-bazel

**Comment by jantoniucci** [comment]
Yes, but it does not work with Bazel 1+ due to a deprecated rule (maven_jar)

Are there plans to support open api generator ? They have more platforms supported like quarkus, micronaut, etc
